cmd/kubeadm/app/constants/constants.go

```diff
@@ -33,11 +33,11 @@ import (
 	"k8s.io/kubernetes/pkg/registry/core/service/ipallocator"
 )
 
-// KubernetesDir is the directory Kubernetes owns for storing various configuration files
-// This semi-constant MUST NOT be modified during runtime. It's a variable solely for use in unit testing.
-var KubernetesDir = "/etc/kubernetes"
-
 const (
+	// KubernetesDir is the directory Kubernetes owns for storing various configuration files
+	// This semi-constant MUST NOT be modified during runtime. It's a variable solely for use in unit testing.
```

Comment:
we should remove the last two sentences:
`// This semi-constant MUST NOT be modified during runtime. It's a variable solely for use in unit testing.`